src/js/ctrl/signals/defaults.ts

```diff
@@ -1,87 +1,98 @@
-'use strict';
-var dl = require('datalib'),
-    ns = require('../../util/ns'),
-    signals = {};
+// TODO(jzong): this probably should be moved to store/factory
```

Comment:
I'm not so sure. This file defines the Vega spec for the signals we need. The store holds the _values_ for the actual signals (i.e., what gets returned if you call Vega's view api `view.signal('name')`).